src/pages/sighting/SightingCore.jsx

```diff
@@ -0,0 +1,279 @@
+import React, { useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useIntl, FormattedMessage } from 'react-intl';
+import { get, flatten, every } from 'lodash-es';
+import { useQueryClient } from 'react-query';
+
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+// import SpeciesIcon from '@material-ui/icons/Category';
+// import RegionIcon from '@material-ui/icons/MyLocation';
+// import ContextIcon from '@material-ui/icons/NaturePeople';
+// import SubmitterIcon from '@material-ui/icons/Person';
+
+import {
+  getAGSQueryKey,
+  getSightingQueryKey,
+} from '../../constants/queryKeys';
+import defaultSightingSrc from '../../assets/defaultSighting.png';
+import MainColumn from '../../components/MainColumn';
+import Link from '../../components/Link';
+import Text from '../../components/Text';
+import LoadingScreen from '../../components/LoadingScreen';
+import SadScreen from '../../components/SadScreen';
+import Button from '../../components/Button';
+import Alert from '../../components/Alert';
+import MoreMenu from '../../components/MoreMenu';
+import ConfirmDelete from '../../components/ConfirmDelete';
+import EntityHeaderNew from '../../components/EntityHeaderNew';
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useDeleteSighting from '../../models/sighting/useDeleteSighting';
+import useCommitAssetGroupSighting from '../../models/assetGroupSighting/useCommitAssetGroupSighting';
+import useSightingFieldSchemas from '../../models/sighting/useSightingFieldSchemas';
+import { formatDate } from '../../utils/formatters';
+// import AnnotationsGallery from './AnnotationsGallery';
+// import IndividualsGallery from './IndividualsGallery';
+import Annotations from './Annotations';
+import Photographs from './Photographs';
+import OverviewContent from './OverviewContent';
+import SightingHistoryDialog from './SightingHistoryDialog';
+import FeaturedPhoto from './featuredPhoto/FeaturedPhoto';
+import Encounters from './encounters/Encounters';
+
+export default function SightingCore({
```

Comment:
note that not all this code is new - most of it is moved from the old SightingCore component